### DISCUSSION TITLE
Custom button card: state_display

### DISCUSSION THREAD

**Discussion by redaste** [indomus/forum#569] [HUB: Home Assistant]
Ciao,
per completare la card in foto ho bisogno di concatenare, al valore di temperatura arrotondato alle sole unità, il simbolo dei gradi.
Non riesco a impostare la corretta sintassi in state_display (che utilizzo per  arrotondare lo stato alle unità e nascondere i decimali)
Suggerimenti?
Grazie
![temperatura](https://user-images.githubusercontent.com/78922425/127162230-bde84887-d148-4cbd-a128-a7f257a18bc1.png)

type: custom:button-card
entity: sensor.tado_temperature
state_display: |
  [[[
	return Math.round(Number(entity.state)).toFixed(0)
  ]]]
name: Temperatura

**Comment by 50494554524F** [comment]
return Math.round(Number(entity.state)).toFixed(0) + ' °C'
